pandas/tests/window/moments/test_moments_ewm.py

```diff
@@ -263,49 +265,68 @@ class TestEwmMomentsConsistency(ConsistencyBase):
     def setup_method(self, method):
         self._create_data()
 
-    def test_ewmcov(self):
-        self._check_binary_ew("cov")
+    def _create_binary_ew_data(self):
+        A = Series(randn(50), index=np.arange(50))
+        B = A[2:] + randn(48)
+
+        A[:10] = np.NaN
+        B[-10:] = np.NaN
+        return A, B
+
+    @pytest.mark.parametrize("min_periods", [0, 1, 2])
+    def test_ewmcov(self, min_periods):
+        A, B = self._create_binary_ew_data()
+
+        self._check_binary_ew(name="cov", A=A, B=B)
+        self._check_binary_ew_min_periods("cov", min_periods, A, B)
 
     def test_ewmcov_pairwise(self):
         self._check_pairwise_moment("ewm", "cov", span=10, min_periods=5)
 
-    def test_ewmcorr(self):
-        self._check_binary_ew("corr")
+    @pytest.mark.parametrize("min_periods", [0, 1, 2])
+    def test_ewmcorr(self, min_periods):
+        A, B = self._create_binary_ew_data()
+
+        self._check_binary_ew(name="corr", A=A, B=B)
+        self._check_binary_ew_min_periods("corr", min_periods, A, B)
 
     def test_ewmcorr_pairwise(self):
         self._check_pairwise_moment("ewm", "corr", span=10, min_periods=5)
 
-    def _check_binary_ew(self, name):
+    def _check_binary_ew(self, name, A, B):
         def func(A, B, com, **kwargs):
             return getattr(A.ewm(com, **kwargs), name)(B)
 
-        A = Series(randn(50), index=np.arange(50))
-        B = A[2:] + randn(48)
-
-        A[:10] = np.NaN
-        B[-10:] = np.NaN
-
         result = func(A, B, 20, min_periods=5)
         assert np.isnan(result.values[:14]).all()
         assert not np.isnan(result.values[14:]).any()
 
+    def _check_binary_ew_min_periods(self, name, min_periods, A, B):
```

Comment:
same

Reply:
changed

Reply:
this does not need to be on the class, it can just be a free function in common.py

also rename these to something like

check_binary_ew_min_periods (de-prevatize)

Reply:
yeah, i fixed! and also combined two methods with parametrization since they are almost identical!

thanks!